packages/connect-web-bench/README.md

```diff
@@ -10,5 +10,5 @@ it like a web server would usually do.
 
 | code generator | bundle size        | minified               | compressed           |
 |----------------|-------------------:|-----------------------:|---------------------:|
-| connect-web    | 87,712 b | 40,923 b | 11,344 b |
+| connect-web    | 87,899 b | 40,992 b | 11,384 b |
```

Comment:
Not sure why the bench numbers changed as a result of this.  Maybe this is related to an earlier change?

Reply:
Yes, forgot to do it in https://github.com/bufbuild/connect-web/pull/272, thanks for the update.